recipes-ofdpa/ofdpa/ofdpa-platform/platform/arm-accton-as4610-30-r0/config.bcm

```diff
@@ -0,0 +1,126 @@
+pbmp_xport_ge=0x0000001fffffe
```

Comment:
This file (and other files for ARM switches) are installed on the generic x86 image. Is that intentional? They space is negligible, it just looks a bit odd.

Reply:
Yes, I didn't consider the space saved worth the effort.